fe/fe-core/src/main/java/com/starrocks/connector/hadoop

```diff
@@ -0,0 +1 @@
+../../../../../../../../java-extensions/hadoop-ext/src/main/java/com/starrocks/connector/hadoop
```

Comment:
what's this

Reply:
symlink, to share code between FE and BE.